icu4c/source/i18n/messageformat2_parser.cpp

```diff
@@ -125,7 +125,13 @@ static bool isContentChar(UChar32 c) {
            || inRange(c, 0xE000, 0x10FFFF);
 }
 
-// See `s` in the MessageFormat 2 grammar
+// See `bidi` in the MF2 grammar
+static bool isBidi(UChar32 c) {
```

Comment:
maybe isBidiControl might be better?

Reply:
Done in [780a947](https://github.com/unicode-org/icu/pull/3236/commits/780a947d1a44d6705a2948ed0ff77504af17dc40)